packages/core/src/test/awsService/accessanalyzer/iamPolicyChecks.test.ts

```diff
@@ -155,6 +155,8 @@ describe('validatePolicy', function () {
                 'us-east-1',
```

Comment:
Can we add a test that will help us determine if there is any change to the expected format in the `getProfileName()` function?

Reply:
The CLI tool tests check for a profile to be passed as `undefined`. This will check if the 'profile:' part gets changed

Reply:
Had a chat with Kevin offline regarding if there is a way for us to catch the change in the format of the profile name, and he confirmed that it should be caught in the unit tests.